www/pages/_document.tsx

```diff
@@ -62,6 +62,7 @@ export default class MyDocument extends Document<Props> {
         </head>
         <body>
           <Main />
+          <NextScript />
 
```

Comment:
with this included, the react app will properly boot on the client, for example in dev mode the React tab was never shown inside the chrome debugger

Reply:
ahh, omitting this is intentional. We don't actually need any client side javascript other than https://github.com/hanford/trends/blob/e141ad37e42ec8aaa0b3aa02d7f43f28a0559867/www/pages/_document.tsx#L98

We're only using react server side to generate the templates, and there is only a few lines of client side javascript aka no react

Reply:
including react would make a perfect performance audit much harder, potentially  impossible

Reply:
the only client side javascript registered a event listener to submit a form when changing input values, and registers a service worker for offline :)

Reply:
cool if it's intentional I am happy to close this PR, I was playing with the router and just rerunning the query on the client side when changing the language from the input and realised React wasn't properly booted